doc/mgr/dashboard.rst

```diff
@@ -11,5 +11,13 @@ configuration key facility. So we can configure them like::
   ceph config-key put mgr/dashboard/$name/server_port $PORT
 
 where ``$name`` is the ID of the ceph-mgr who is hosting this dashboard web app.
-If they are not configured, the web app will be bound to ``127.0.0.1:7000``.
```

Comment:
Yes, in 015bc3f5c59353e24a367ce3aba7fd62b6f44afc this was reverted and the manager will throw a RunTimeError when no address has been configured